### ISSUE TITLE
NTR: plant secondary metabolite 

### ISSUE THREAD

**Issue by LilyAndres** [Southern-Cross-Plant-Science/cdno#29]
## Preferred term label

plant secondary metabolite

## Textual definition

A group of naturally occurring compound classes biosynthesized by differing biochemical pathways whose plant content and regulation is strongly susceptible to environmental influences and to potential herbal predators.

[Pavarini D. et al., 2012](https://www.sciencedirect.com/science/article/pii/S0377840112002350)

## Suggested parent term

Parent CDNO 'dietary nutritional component'

**Comment by LilyAndres** [comment]
Maybe we can make 'plant secondary metabolite' a subclass of plant metabolite CHEBI:76924, hence the definition would be:

A plant metabolite [CHEBI:76924] that is not essential for growth or division of plant cells.

**Comment by LilyAndres** [comment]
Hey @kaiiam, I would like to know your opinion on this please, it's related to the role in CHEBI again.

One of the roles from CHEBI, the term ['secondary metabolite' [CHEBI:26619]](https://www.ebi.ac.uk/ols/ontologies/chiro/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FCHEBI_26619&viewMode=All&siblings=false) doesn't appear in CHEBI webpage but it appears in [OLS].

And in CHEBI webpage, the term ['plant metabolite' [CHEBI:76924]](https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:76924) has synonym 'plant secondary metabolites' 

It's a bit confusing, but I think we could use 'secondary metabolite' [CHEBI:26619] from [OLS](https://www.ebi.ac.uk/ols/ontologies/chiro/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FCHEBI_26619&viewMode=All&siblings=false)

Where the definition is:
"A metabolite that is not directly involved in the normal growth, development or reproduction of an organism". 

Or create 'create secondary metabolite' in CDNO.

**Comment by kaiiam** [comment]
Interesting @LilyAndres the above links are not from CHEBI but is instead from CHIRO https://github.com/obophenotype/chiro the chebi integrated role ontology, looks like attempts by Chris Mungall and others to connect CHEBI roles more explicitly to other OBO ontologies. I didn't know about CHIRO. Looks like they grabbed everything under role from chebi and are trying to fix/align them with other ontologies. 

The CHIRO term 'secondary metabolite' [CHEBI:26619] redirects to the chebi term [CHEBI:25212 - metabolite](https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:26619) with CHEBI:26619 as a secondary ID. And as you say  [CHEBI:76924 - plant metabolite](https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:76924) has syn `plant secondary metabolites`. Both classes are under role. You'd think that a new term `plant secondary metabolite` should be subclass to plant metabolite. 

Chebi's [metabolite](https://www.ebi.ac.uk/ols/ontologies/chebi/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FCHEBI_25212)  has  the def: 

> Any intermediate or product resulting from metabolism. The term 'metabolite' subsumes the classes commonly known as primary and secondary metabolites.

I have no idea why CHRIO has CHEBI:26619 but CHEBI doesn't, however, maybe the thing to do for now for our purposes is to ask to add or yourself add to CHEBI the term `plant secondary metabolite` as subclass to CHEBI [plant metabolite](https://www.ebi.ac.uk/ols/ontologies/chebi/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FCHEBI_76924&viewMode=All&siblings=false).

**Comment by LilyAndres** [comment]
Ok, so I have an idea. It has been highlighted that the secondary metabolites are also called 'specialized metabolites' [(1)](https://science.sciencemag.org/content/311/5762/808.full), [(2)](https://www.sciencedirect.com/science/article/pii/S0168945220301850#bib0005), [(3)](https://www.sciencedirect.com/science/article/pii/S0958166917301647), [(4)](https://www.frontiersin.org/articles/10.3389/fmicb.2019.02550/full), [(5)](https://en.wikipedia.org/wiki/Secondary_metabolism#:~:text=Secondary%20metabolism%20(also%20called%20specialized,the%20survival%20of%20the%20organism.),)
If we use this term I can have it ready in CHEBI in a few minutes, being a subclass of 'eukaryotic metabolite' [CHEBI:75763]

**Comment by kaiiam** [comment]
CHEBI:75763 is [eukaryotic metabolite](https://www.ebi.ac.uk/ols/ontologies/chebi/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FCHEBI_75763), which has as subclass plant metabolite. My suggestion is to make the term `plant secondary metabolite` as subclass to [plant metabolite](https://www.ebi.ac.uk/ols/ontologies/chebi/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FCHEBI_76924&viewMode=All&siblings=false). That fits with my understanding of the theory you're describing.  

<img width="333" alt="image" src="https://user-images.githubusercontent.com/12255688/105507324-0c3ce080-5c99-11eb-9eae-5bfaaa8d3397.png">

**Comment by LilyAndres** [comment]
Ok, so if it's okay I think it could be of value to have this term in CDNO rather than CHEBI. Adnan has expressed that 

>"terms used interchangeably in nutrition science will lead to more confusion among CHEBI users since ChEBI is a chemistry database and the introduction of additional terms will make the ChEBI ontology more complex for their users who come from various domains" (email).

So I suggest we create it in CDNO and later if they think it's necessary in CHEBI, they can take it. 

If this is ok then we will have the label `plant secondary metabolite` with the synonym `secondary metabolite` and the definition:

A [plant metabolite [CHEBI:76924]](https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:76924) that is not essential for growth or division of plant cells.

**Comment by kaiiam** [comment]
Sounds reasonable, were just trying to uphold [OBO principal 5](http://obofoundry.org/principles/fp-005-delineated-content.html) aka reuse other OBO domain ontologies and don't reinvent the wheel. In this case since Adan said CHEBI might not be appropriate then it should be a nutrition ontology and as far as I know CDNO is the first in OBO (but don't quote me on that and we should double check that). Thus if they want to re-direct terms to a nutrition ontology its in scope for us to pick them up. Sorting this out more generally (aka all chebi terms which should be in a nutrition ontology) is probably a larger overall issue which we might not be able to solve in the short term but we should keep this in mind moving forward. 

@LilyAndres your plan for `plant secondary metabolite` sounds good for now I'd say go for it!

**Comment by LilyAndres** [comment]
Great, thanks @kaiiam. 

Yes, as we have discussed before, CDNO could evolve to be DNO integrating all those terms that might not fit in CHEBI. The nutrition ontologies that exist in OBO are the Ontology for Nutritional Epidemiology (ONE), Ontology for Nutritional Studies (ONS) and Food-Biomarker Ontology (FOBI), but they have a very simple and inconsistent classification of dietary nutritional components. So, at the moment CDNO would be managing this domain, but maybe it can evolve to DNO including other nutritional components derived from other kingdoms and not only from plants. 

I moved the definition of 'plant secondary metabolite' and made the changes in the ROBOT template. I will close this issue now, thanks for all the feedback and help with the creation of the terms :)
